why-apollo-advantages-disadvantages-alternatives.md

```diff
@@ -51,93 +51,90 @@ There are plenty of popular and tech savvy companies out there who are using Apo
 
 {{% sub_chapter_header "Apollo's Documentation" "apollo-documentation" %}}
 
-Even though Apollo is evolving all the time, the team and community behind it makes sure to keep the documentation up to date. Going through their documentation should give you plenty of insights into how you can build your first application with Apollo without reading up any third-party resources. They cover so many areas that it can be also overwhelming for a newcomer. So make sure you go through the content topic by topic and apply it in your own application.
+While Apollo continues to evolve, the team and community behind it keeps the documentation up to date, and the there are plenty of insights on how to build that first application. In fact, they cover so many areas it can be overwhelming for newcomers. 
```

Comment:
"the team and community behind it keeps the documentation up to date, and the there are plenty of insights on how to build that first application." <- sentence seems a bit broken

Reply:
It's a bit less text to require a headline for this topic, is it?

Reply:
I don't understand the 2nd comment's question.  A bit less text to require a headline?

Reply:
The editing made it less text now. So it's fewer sentences where I am not sure if they justify their own headline. I guess that's why I have formulated the sentences so bloated before :D